src/bitmap/imp.rs

```diff
@@ -220,7 +220,7 @@ impl Bitmap {
     ///
     /// let mut bitmap = Bitmap::create();
     /// bitmap.add(1);
-    /// bitmap.remove(1);
+    /// bitmap.remove(&1);
```

Comment:
Any reason for this change?

Reply:
Ah, it's a mistake, I was trying to change the argument of ``remove`` to reference, just like other rust collections, but then I think it's too small to consider.

Reply:
Could you please remove this change then. Thank you!

Reply:
Just force pushed it ;D

Reply:
It's fine, thank you!